content/direct-deposit-management/release-notes/2023-09-06.md

```diff
@@ -0,0 +1,15 @@
+The API Team has updated error handling for the API. All backend system timeouts are increased to 17 seconds
+New error message added for timeouts related to one of the backends
```

Comment:
What is happening to the backend here? We'll need a little for specificity? Is there a disconnection with a backend system? 
```suggestion
- New error message added for timeouts related to one of the backends
```